app/Http/Controllers/Multiplayer/Rooms/Playlist/ScoresController.php

```diff
@@ -70,7 +71,7 @@ public function index($roomId, $playlistId)
             $userHighScore = $playlist->highScores()->where('user_id', $user->getKey())->first();
 
             if ($userHighScore !== null) {
-                $userScoreJson = json_item($userHighScore->score, 'Multiplayer\Score', ScoreTransformer::BASE_INCLUDES);
+                $userScoreJson = json_item($userHighScore->score, $transformer, ScoreTransformer::BASE_INCLUDES);
```

Comment:
`MULTIPLAYER_BASE_INCLUDES`, also should use `->scoreLink` and check for null (and probably have this condition in a test)